### PR TITLE
Remove DocuSign icon

### LINKED ISSUE #11230: Update: DocuSign
### Brand Name

DocuSign

### Official Resources for Icon and Color

DocuSign got a new logo on April 16, 2024.
https://www.docusign.com/en-sg/blog/next-brand-chapter

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by anhthang**
image

**Issue:** closes #11230 

**Popularity metric:**

Global Rank 1,229

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

### CODE REVIEW

**Comment by uncenter** [comment]
It also looks like there brand name is (now?) "Docusign" (lowercase S). Would you mind updating @anhthang?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4389,8 +4389,9 @@
         },
         {
             "title": "DocuSign",
-            "hex": "FFCC22",
-            "source": "https://www.docusign.com/sites/all/themes/custom/docusign/favicons/mstile-310x310.png"
+            "hex": "4D00FC",
```
Comment (uncenter) [inline _data/simple-icons.json:4392]: Where did you get this color from?

File: _data/simple-icons.json [hunk 2]
```diff
@@ -4389,8 +4389,9 @@
...
+            "source": "https://brandfolder.com/docusign-1/press-kit",
+            "guidelines": "https://brandfolder.com/docusign-1/press-kit"
```
Comment (uncenter) [inline _data/simple-icons.json:4394]:
```suggestion
            "source": "https://brand.docusign.com",
            "guidelines": "https://brand.docusign.com"
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (uncenter) [reply _data/simple-icons.json:4392]: (I'm seeing `#4c00ff` from the color section of their brand guidelines.)

Comment (anhthang) [reply _data/simple-icons.json]: cannot remember where i copied. just updated following their brand palette

File: icons/docusign.svg [hunk 3]
```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>DocuSign</title><path d="M9.517 3.31h4.966v6.621h3.31L12 16.552 6.207 9.931h3.31V3.31zM0 19.034h24v1.655H0v-1.655z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>DocuSign</title><path d="M7.298.006a.582.582 0 0 0-.583.583v6.12h6.493a.692.692 0 0 1 .496.21l3.37 3.371c.136.124.21.31.21.496v6.492C20.99 17.266 24 13.784 24 8.63c0-5.167-3.308-8.636-7.62-8.624Zm9.987 17.272h-10a.582.582 0 0 1-.582-.582V6.722H.583A.582.582 0 0 0 0 7.304v16.108c0 .322.26.582.582.582h16.12c.322 0 .583-.26.583-.582z"/></svg>
```
Comment (uncenter) [inline icons/docusign.svg:1]: You'll need to update the capitalization here as well, sorry.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/docusign.svg (hunk 3 above)
Comment (anhthang) [reply icons/docusign.svg:1]: oh yes, i forgot to update this

**Comment by adamrusted** [comment]
Page 27 of [this document](https://cdn.bfldr.com/HIN7H4LI/at/pk9tfh4hv68x9fr8w9gbf4fx/Docusign_Brand_Guidelines-Public.pdf) states:

> You may not ... distribute or otherwise make available our logos, marks, or assets

We will therefore need to remove Docusign from the collection.

**Comment by anhthang** [comment]
@adamrusted you wanna convert this PR to removal, right?

**Comment by adamrusted** [comment]
Yeah, was just asking if you were okay to do the removal in this PR or whether you wanted us to close this, and remove it elsewhere.

**Comment by anhthang** [comment]
No problem, I've removed the icon and updated the PR title for clarity.

**Comment by adamrusted** [comment]
Awesome, thanks @anhthang - we've got a release at the end of this month (Microsoft removal) so can look to include there :)

**Pull request merged**